### ISSUE TITLE
Analog movement broken

### ISSUE THREAD

**Issue by colegmr** [pa1nki113r/Project_Brutality#235]
When using a controller (I use one bc my wrist is busted) analog movement no longer works 
after the most recent patch you run at full speed or not at all. I have gone through all options and am at a loss.

P.S. I love this mod thank you for making it.

**Comment by scaryred24** [comment]
Zmovement is far from complete and is still in the works. As that is being updated, we have to play the cat and mouse game to catch up with it. As for controller support, personally I could never wrap my head around such a complex control scheme on a controller. Those that are handicapped will eventually find a way for it to work as each one is unique from the other. I feel bad that you have a wrist injury that prevents you from play via keyboard and mouse.

**Comment by colegmr** [comment]
Thanks for the reply and good luck.

**Comment by colegmr** [comment]
So analog movement is still bugged maybe adding a toggle in the options to enable/disable zmove entirely would fix it 
if that's even doable 
I imagine it would be a lot of work 
but it would solve a lot of issues ppl are having with it until the kinks are ironed out.

**Comment by JMartinez9820** [comment]
Unfortunately that might be impossible do to ZMovement now being hardcoded into PB. I would try ZMovement just by itself to see if analog movement is still broken, you can get it by itself on ZDoom Forums, just Google up ZMovement.

**Comment by colegmr** [comment]
thx Ill let you know.

**Comment by colegmr** [comment]
Yep its zmove, that's a shame their zdoom page says this is most likely their final version too.

**Comment by popguy12** [comment]
Can we get a double check on whether this is fixed or fixed in the newer version of ZMove?

**Comment by Bynbleidd** [comment]
Not fixed, still full speed or nothing.
Sry, I misread. Dont know about zmove, biut its definitely no movement change achievable via stick in PB 04.

**Comment by MasterZeroFlash** [comment]
Yea, I just tried PB for the first time in forever (love the mod btw), and using an Xbox One controller precise movement is just about impossible it feels like, with run on I move way too fast and bump into everything, run off seems better though but then I move a bit too slow. Tried fiddling with the Analogue Run/Strafe options but I dont think those options did much more than what can be attributed to placebo (or I sometimes felt a difference, but couldnt for the life of me say what), but in any case it is pretty awful to play with a controller at this point. 

Keyboard worked a lot better, but thats not really my thing. 

Is something on the cards to have a look into this at some point in the future or does anyone at least know of any workaround or custom patch or such (that isnt just "Play with kb+m instead", heh).

TLDR: Just tried PB. Xbox One controller. Movement is pretty horrible.

**Comment by nemesis-vampy** [comment]
was this on 0.4.1 or staging branch? in staging this problem should be fixed, if it's not i'll need to figure out how to reproduce it since it doesn't happen for me.

**Comment by MasterZeroFlash** [comment]
Staging Branch is when you download the latest source code, right? If so, then yes, it was on that. Gonna try with a clean .ini file just to see what happens as well.

**Comment by nemesis-vampy** [comment]
if it says 0.4.2 in the pb options menu it's staging

**Comment by MasterZeroFlash** [comment]
> if it says 0.4.2 in the pb options menu it's staging

It fixed! Fantastico!

What are the Analog Run/Strafe options supposed to do btw? I dont seem to really notice much of a difference.

**Comment by nemesis-vampy** [comment]
i've no idea honestly, outside of doing some testing to make sure some pb features work with it i've not really used a controller with uzdoom. the wiki might have info on it, though.

**Comment by MasterZeroFlash** [comment]
> i've no idea honestly, outside of doing some testing to make sure some pb features work with it i've not really used a controller with uzdoom. the wiki might have info on it, though.

Allright. I can for the life of me not decide if it makes a difference or not anyway (maybe it is not for controllers at all?), but everything is much much better now, Ill only need to do some regular fine-tuning :) 

Still have a little bit of that weirdness where it feels kind of like the windows for moving diagonally vs straight forward/back is too big somehow, but it works well enough now!
